pkg/vm/contract_checks.go

```diff
@@ -36,65 +36,65 @@ func getNumOfThingsFromInstr(instr opcode.Opcode, param []byte) (int, bool) {
 // IsMultiSigContract checks whether the passed script is a multi-signature
 // contract.
 func IsMultiSigContract(script []byte) bool {
-	_, ok := ParseMultiSigContract(script)
+	_, _, _, ok := ParseMultiSigContract(script)
 	return ok
 }
 
 // ParseMultiSigContract returns list of public keys from the verification
 // script of the contract.
-func ParseMultiSigContract(script []byte) ([][]byte, bool) {
+func ParseMultiSigContract(script []byte) (int, int, [][]byte, bool) {
```

Comment:
1) Comment says something different
2) Returning the number of keys seems to be redundant as it's a `len(pubs)`.